drivers/delta_ups-mib.c

```diff
@@ -31,39 +31,43 @@
 
 /* To create a value lookup structure (as needed on the 2nd line of the example
  * below), use the following kind of declaration, outside of the present snmp_info_t[]:
- * static info_lkp_t delta_ups_onbatt_info[] = {
- * 	{ 1, "OB", NULL, NULL },
- * 	{ 2, "OL", NULL, NULL },
- * 	{ 0, NULL, NULL, NULL }
+ * static info_lkp_t delta_onbatt_info[] = {
+ * 	info_lkp_default(1, "OB"),
+ * 	info_lkp_default(2, "OL"),
+ * 	info_lkp_sentinel
  * };
  */
 
 static info_lkp_t delta_ups_upstype_info[] = {
-	{ 1, "on-line", NULL, NULL },
-	{ 2, "off-line", NULL, NULL },
-	{ 3, "line-interactive", NULL, NULL },
-	{ 4, "3phase", NULL, NULL },
-	{ 5, "splite-phase", NULL, NULL },
-	{ 0, NULL, NULL, NULL }
+	info_lkp_default(1, "on-line"),
+	info_lkp_default(2, "off-line"),
+	info_lkp_default(3, "line-interactive"),
+	info_lkp_default(4, "3phase"),
+	info_lkp_default(5, "splite-phase"),
```

Comment:
@aquette : is it "splite" (some vendor key word?) or "split" phase here? Any ideas? :)

Reply:
Offloaded to #2302